content/blog/2019/11/2019-11-17-gsoc-report.adoc

```diff
@@ -0,0 +1,217 @@
+
+Nguyen Le Vu Long presented the 2.0 release of the Remoting over Apache Kafka project. In particular, the Kafka launcher, Cloud and Helm chart were presented. Long also talked about the new improvements in Phase 3.
+
+Links:
+
+* link:https://jenkins.io/projects/gsoc/2019/remoting-over-apache-kafka-docker-k8s-features[Project page]
+* link:https://docs.google.com/presentation/d/1QMjeAt6on3RPQdjlz4SiFg8YgpJ2xNOQhfzXDWgn2AY/[Link to coding phase 3 presentation]
+* link:https://jenkins.io/blog/2019/08/19/remoting-kafka-kubernetes-release-2/[Blog post for release 2 of the plugin]
+* link:https://github.com/jenkinsci/remoting-kafka-plugin[Source code]
+
+== Multi-branch Pipeline support for Gitlab
+
+* *Student*: link:/blog/authors/baymac[Parichay Barpanda]
+* *Lead mentors*: link:/blog/authors/markyjackson-taulia[Marky Jackson-Taulia], link:/blog/authors/justinharringa[Justin Harringa]
+* *Goal*: Add Multibranch Pipeline support for GitLab SCM private and SaaS instances so that Jenkins can automatically build branches and pull requests with Jenkinsfile.
+
+Parichay presented the new GitLab Branch Source Plugin with Multibranch Pipeline Jobs and Folder Organisation support.
+The presentation starts with a brief recap of the work done in Phase 1 and the new features implemented in Phase 2.
+A demo on how to setup the plugin and using it on your Jenkins instance.
+It concludes by inviting contributions to the project and presents the roadmap planned. 
+
+Links:
+
+* link:https://jenkins.io/projects/gsoc/2019/gitlab-support-for-multibranch-pipeline[Project page]
+* link:https://docs.google.com/presentation/d/1avMoEPMf5Ksc99106fv7bGikAh17GQZJZdwUWKOzYBY/[Link to coding phase 3 presentation]
+* link:https://github.com/jenkinsci/gitlab-branch-source-plugin/[Source code]
+
+== Recognition
+
+Last year, we wanted to thank everyone who participated in the program by sending swag.
+This year, we collected all the mailing addresses we could and sent to everyone we could the 15-year Jenkins special edition T-shirt, and some stickers.
+This was a great feel good moment.
+I want to personally thank Alyssa Tong her help on setting aside the t-shirt and stickers.
+
+image:/images/post-images/2019-11-17-gsoc-report/swag-before-shipping.jpg[width=400]
+
+We try to get our successful students to conferences when possible and when the budget allows it.
+We were able to send one student to the
+link:https://www.cloudbees.com/devops-world/san-francisco[DevOps World | Jenkins World San Francisco] conference.
+We are working on sending students to the link:https://www.cloudbees.com/devops-world/lisbon[DWJW conference] in Lisbon, Portugal.
+
+== Conclusion
+
+This year, five projects were successfully completed. We find this to be normal an in line with what we hear from other participating organizations.
+
+Taking the time early to update our GSoC pages saved us a lot of time later on because we did not have to repeat all the information every time someone contacted us.
```

Comment:
```suggestion
Taking the time early to update our GSoC pages saved us a lot of time later because we did not have to repeat all the information every time someone contacted us.
```

Reply:
LGTM